packages/localstorage/localstorage.js

```diff
@@ -21,7 +21,33 @@ try {
 } catch (ignored) {}
 
 if (key === retrieved) {
-  Meteor._localStorage = storage;
+  if (Meteor.isServer) {
+    Meteor._localStorage = storage;
+  } else {
+    // IE11 doesn't handle properly attempts to change methods of the
```

Comment:
This comment is a bit verbose, and needs a few grammatical tweaks. How about something like this instead?

```
// Some browsers (e.g. IE11) don't properly handle attempts to change 
// window.localStorage methods. By using proxy methods to expose 
// window.localStorage functionality, developers can change 
// the behavior of Meteor._localStorage methods without breaking 
// window.localStorage.
```